### DISCUSSION TITLE
imagery

### DISCUSSION THREAD

**Discussion by alexhoffenreich** [nasa/openmct#7443] [Community Q&A] [answered]
How can I disable the thumbnail trip in the imagery plug-in?

**Comment by ozyx** [comment, marked as answer]
Hi Alex,

There is currently no built-in way to disable the thumbnail strip in the imagery view (feel free to [file a feature request](https://github.com/nasa/openmct/issues/new?assignees=&labels=type%3Aenhancement&projects=&template=enhancement-request.md&title=), though!). However, thumbnail strips will disappear if the view is scaled down enough (such as when embedded in a Display Layout).

Example:

![image](https://github.com/nasa/openmct/assets/9259993/54d1533e-30ad-4a43-a8a2-d5e2286c1af7)

Would this solution serve your purposes?

**Reply by alexhoffenreich** [reply]
Thanks, 
Does it mean that it will also not save thumbs in memory?
I want to focus my problem.
I don't need to hide the thumbs
only to limit their number to the range visible in the current viewport 
My problem is that I am feeding the video from a live feed at 30hz and the plug-in stores all those thumbs in memory and doesn't seem to free the memory once they are not visible 
Maybe I'm doing something wrong....
I appreciate any help you can offer.
